src/NUnitFramework/framework/Constraints/DictionaryContainsKeyConstraint.cs

```diff
@@ -33,6 +37,8 @@ namespace NUnit.Framework.Constraints
     /// </summary>
     public class DictionaryContainsKeyConstraint : CollectionItemsEqualConstraint
     {
+        private bool _deprecatedFlag = false;
```

Comment:
I'd rather put 'is' in the name of a bool property than 'flag.' `_isLegacyMode` or `_isDeprecatedMode`?